### DISCUSSION TITLE
为什么获取配置的url没有环境参数？

### DISCUSSION THREAD

**Discussion by lovebryant24** [apolloconfig/apollo#4340] [Q&A] [answered]
为什么获取配置的url没有环境参数？
https://www.apolloconfig.com/#/zh/usage/other-language-client-user-guide
![image](https://user-images.githubusercontent.com/19700611/167236953-0a485c40-e35d-4509-8972-2864faa519e4.png)

**Comment by nobodyiam** [comment, marked as answer]
Each environment will have a different config_server_url.

**Reply by lovebryant24** [reply]
thanks
